terminus-ui/expansion-panel/testing/src/test-helpers.ts

```diff
@@ -0,0 +1,263 @@
+  const triggerDebugElement = panelDebugElement.query(By.css('ts-expansion-panel-trigger'));
+  if (!triggerDebugElement) {
+    throw new Error(`getTriggerDebugElement found no triggers within the panel at index '${panelIndex}'`);
+  }
+  return triggerDebugElement;
+}
+
+/**
+ * Get the instance for a TsExpansionPanelTriggerComponent
+ *
+ * @param fixture - The component fixture
+ * @param index - The index of the desired TsExpansionPanelComponent
+ * @return The instance
+ */
+export function getTriggerInstance(fixture: ComponentFixture<any>, panelIndex = 0): TsExpansionPanelTriggerComponent {
+  const triggerDebugElement = getTriggerDebugElement(fixture, panelIndex);
+  return triggerDebugElement.componentInstance;
+}
+
+/**
+ * Get the element for a TsExpansionPanelTriggerComponent
+ *
+ * @param fixture - The component fixture
+ * @param index - The index of the desired TsExpansionPanelComponent
+ * @return The element
+ */
+export function getTriggerElement(fixture: ComponentFixture<any>, panelIndex = 0): HTMLElement {
+  const triggerDebugElement = getTriggerDebugElement(fixture, panelIndex);
+  return triggerDebugElement.nativeElement;
+}
+
+/**
+ * Get the element for a TsExpansionPanelTriggerTitleComponent
+ *
+ * @param fixture - The component fixture
+ * @param index - The index of the desired TsExpansionPanelComponent
+ * @return The element
+ */
+export function getTriggerTitleElement(fixture: ComponentFixture<any>, panelIndex = 0): HTMLElement {
+  const triggerDebugElement = getTriggerDebugElement(fixture, panelIndex);
+  const title = triggerDebugElement.query(By.css('.ts-expansion-panel__trigger-title'));
+  if (!title) {
+    throw new Error(`getTriggerTitleElement found no title within the trigger at index '${panelIndex}'`);
+  }
+  return title.nativeElement;
+}
+
+/**
+ * Get the element for a TsExpansionPanelTriggerDescriptionComponent
+ *
+ * @param fixture - The component fixture
+ * @param index - The index of the desired TsExpansionPanelComponent
+ * @return The element
+ */
+export function getTriggerDescriptionElement(fixture: ComponentFixture<any>, panelIndex = 0): HTMLElement {
+  const triggerDebugElement = getTriggerDebugElement(fixture, panelIndex);
+  const description = triggerDebugElement.query(By.css('.ts-expansion-panel__trigger-description'));
+  if (!description) {
+    throw new Error(`getTriggerTitleElement found no description within the trigger at index '${panelIndex}'`);
+  }
+  return description.nativeElement;
+}
+
+/**
+ * Get the element for a TsExpansionPanelActionRowComponent
+ *
+ * @param fixture - The component fixture
+ * @param index - The index of the desired TsExpansionPanelComponent
+ * @return The element
+ */
+export function getPanelActionRow(fixture: ComponentFixture<any>, index = 0): HTMLElement {
+  const debugElement = getPanelDebugElement(fixture, index);
+  const row = debugElement.query(By.css('.ts-expansion-panel__action-row'));
```

Comment:
here's the test helper for TsExpansionPanelActionRowComponent, using the query selector